content/docs/tasks/traffic-management/secure-ingress/index.md

```diff
@@ -0,0 +1,304 @@
+---
+title: Secure Ingress Traffic
```

Comment:
can we clearly name this as Securing Gateways with HTTPS?

Reply:
I would propose `Secure Ingress Gateways with TLS`, since we are talking about configuring  TLS/mTLS for ingress traffic. Or  `Secure Ingress Traffic with TLS`, `Configure TLS for Ingress Traffic` or `Configure TLS for Ingress Gateways`. More ideas are welcome.

@geeknoid @frankbu @rshriram

Reply:
@geeknoid @frankbu @rshriram could you please agree on the title? See https://github.com/istio/istio.github.io/pull/1590#discussion_r198693721 and https://github.com/istio/istio.github.io/pull/1590#discussion_r198668429.

The tests pass, I would like to merge the PR.

Reply:
Sorry, I merged it already. I like Shriram's suggestion: Securing Gateways with HTTPS